.github/workflows/release_publish.yml

```diff
@@ -36,3 +36,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_API_TOKEN: ${{ secrets.NPM_API_TOKEN }}
+      - name: Upload Standalone Bundle Asset
```

Comment:
I haven't tested this but I think it should work... Obviously we'll have to iterate on it